javascript/index.js

```diff
@@ -3,38 +3,106 @@
 
 // 🚨🚨🚨 Comment out the below code before you start working on the code
 
-// Out of sync
-  getInstruction("mashedPotatoes", 0, (step1) => {
+// Iteration 1 - using callbacks
+getInstruction("mashedPotatoes", 0, (step0) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step0}</li>`;
+  // ... Your code here
+  getInstruction("mashedPotatoes", 1, (step1) => {
     document.querySelector("#mashedPotatoes").innerHTML += `<li>${step1}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 1, (step2) => {
-    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step2}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 2, (step3) => {
-    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step3}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 3, (step4) => {
-    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step4}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 4, (step5) => {
-    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
-    document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
-  }, (error) => console.log(error));
 
+    getInstruction("mashedPotatoes", 2, (step2) => {
+      document.querySelector("#mashedPotatoes").innerHTML += `<li>${step2}</li>`;
 
+      getInstruction("mashedPotatoes", 3, (step3) => {
+        document.querySelector("#mashedPotatoes").innerHTML += `<li>${step3}</li>`;
 
-// Iteration 1 - using callbacks
-// ...
+        getInstruction("mashedPotatoes", 4, (step4) => {
+          document.querySelector("#mashedPotatoes").innerHTML += `<li>${step4}</li>`;
+          document.querySelector("#mashedPotatoes").innerHTML += `<li>Mashed potatoes are ready!</li>`;
+
+          document.getElementById("mashedPotatoesImg").removeAttribute("hidden");
+        });
```

Comment:
Here, don't forget to catch your errors. 
Each time, before you chain another callback, you should deal with the error messages on your console.
```
getInstruction(
  'mashedPotatoes',
  0,
  (step0) => {
    document.querySelector('#mashedPotatoes').innerHTML += `<li>${step0}</li>`;
    getInstruction(
      'mashedPotatoes',
      1,
      (step1) => {
        document.querySelector('#mashedPotatoes').innerHTML += `<li>${step1}</li>`;
        getInstruction(
          'mashedPotatoes',
          2,
          (step2) => {
            document.querySelector('#mashedPotatoes').innerHTML += `<li>${step2}</li>`;

(...)

         },
          (error) => console.log(error)
        );
      },
      (error) => console.log(error)
    );
  },
  (error) => console.log(error)
);
```